### PR TITLE
Fix ngeo examples

### PR DESCRIPTION
**Pull request by llienher**
For now, following examples are fixed in this PR:
- datepicker
- displaywindow
- layerorder
- backgrounddropdown
- rotate
- scaleselector
- colorpicker
- drawFeature
- googlestreetview
- search
- layertree

### CODE REVIEW

File: examples/backgroundlayerdropdown.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ import olLayerImage from 'ol/layer/Image.js';
 import olLayerTile from 'ol/layer/Tile.js';
 import olSourceImageWMS from 'ol/source/ImageWMS.js';
 import ngeoMapModule from 'ngeo/map/module.js';
+import '../node_modules/bootstrap/js/dropdown.js';
```
Comment (gberaudo) [inline examples/backgroundlayerdropdown.js]: Should be
```
import 'bootstrap/js/dropdown.js';
```

File: examples/datepicker.js [hunk 2]
```diff
@@ -4,6 +4,7 @@
 const exports = {};
 
 import './datepicker.css';
+import '../node_modules/jquery-ui/themes/base/all.css';
```
Comment (gberaudo) [inline examples/datepicker.js]: Should be
```
import 'jquery-ui/themes/base/all.css';
``

File: examples/datepicker.js [hunk 3]
```diff
@@ -4,6 +4,7 @@
 const exports = {};
 
 import './datepicker.css';
+import 'jquery-ui/themes/all.css';
```
Comment (gberaudo) [inline examples/datepicker.js]: Why do you take all.js here?
Shouldn't the CSS be imported in the source code needing it?

File: examples/scaleselector.js [hunk 4]
```diff
@@ -11,6 +11,7 @@ import olView from 'ol/View.js';
 import olLayerTile from 'ol/layer/Tile.js';
 import olSourceOSM from 'ol/source/OSM.js';
 import ngeoMapModule from 'ngeo/map/module.js';
+import 'bootstrap/js/dropdown.js';
```
Comment (gberaudo) [inline examples/scaleselector.js]: idem

File: buildtools/webpack.commons.js [hunk 5]
```diff
@@ -188,7 +188,7 @@ const config = {
       'ol': 'openlayers/src/ol',
       // For angular-ui-date
       'jquery-ui/datepicker' : 'jquery-ui/ui/widgets/datepicker',
-      'jquery-ui' : 'jquery-ui/ui',
```
[the remark this answers is not in the document]
Comment (gberaudo) [reply buildtools/webpack.commons.js]: Why not?
'jquery-ui/datepicker'is required by angular-ui-date however 'jquery-ui' does not appear to be necessary.

File: buildtools/webpack.commons.js [hunk 6]
```diff
@@ -186,9 +186,7 @@ const config = {
       'jsts': 'jsts/org/locationtech/jts',
       'ol/ol.css': 'openlayers/css/ol.css',
       'ol': 'openlayers/src/ol',
-      // For angular-ui-date
-      'jquery-ui/datepicker' : 'jquery-ui/ui/widgets/datepicker',
-      'jquery-ui' : 'jquery-ui/ui',
+      'jquery-ui/datepicker' : 'jquery-ui/ui/widgets/datepicker',       // For angular-ui-date
```
Comment (sbrunner) [inline buildtools/webpack.commons.js]: Why so many spaces?

**Pull request merged**